doc/source/overview/legacy_kiwi.rst

```diff
@@ -109,6 +109,17 @@ consider to use the legacy KIWI version.
    Many distributions also dropped the lxc tools from the distribution
    in favour of docker.
 
+*  OEM Recovery/Restore
+
+   Recovery/Restore in the world of images has been moved from the OS
+   layer into higher layers. For example in Cloud and Public Cloud
```

Comment:
"Cloud and Public Cloud environments"

What's the difference between "Cloud" and "Public Cloud"? 😉

Either you just write "Cloud environments" or -- if you want to emphasis private and public clouds -- you could write:

"In private and public Cloud environments..."